src/Foundation/Features/Checkout/Checkout.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model CheckoutViewModel
+
+@using (Html.BeginForm("PlaceOrder", "Checkout", FormMethod.Post, new { @class = "row jsCheckoutForm", id = "jsCheckoutForm", novalidate = "novalidate" }))
+{
+    @Html.AntiForgeryToken()
+    <div class="col-md-8 col-xs-12">
+        <div class="card items">
+            <div class="card-body">
+                @Html.Partial("ShippingInformation", Model)
+                <br />
+                @Html.Partial("SingleAddress", Model)
+                <hr />
+                @Html.Partial("BillingInformation", Model)
+            </div>
+        </div>
+        <br />
+        <div class="card items">
+            <div class="card-body">
+                @Html.Partial("Subscription", Model)
```

Comment:
Please add the following to subscriptions

- Change checkbox label to create recurring order.
- Remove cycle mode as we will always use days
- Update the label to and text box for cycle length to be Every (Text Box) days
- Add new datetime textbox for choosing the end date.  Can be nullable
- Update the start date to be x number of days from current order